src/ocean/time/StopWatch.d

```diff
@@ -17,6 +17,9 @@
 
```

Comment:
What's the relevance of the comment "On Linux clock_getres(CLOCK_MONOTONIC) reports a resolution of 1ns."? Maybe explain this a little bit more.